### PR TITLE
[BPK-971] Upgrade webpack deps

### PR DESCRIPTION
**Pull request by matthewdavidson**
This is one of the many PR's aimed at upgrading dependencies in the backpack monorepo. I focus on updating webpack based dependencies. Here are the list of changelogs:

- https://github.com/postcss/autoprefixer/blob/master/CHANGELOG.md
- https://github.com/webpack-contrib/extract-text-webpack-plugin/blob/master/CHANGELOG.md
- https://github.com/webpack-contrib/file-loader/blob/master/CHANGELOG.md
- https://github.com/postcss/postcss-loader/blob/master/CHANGELOG.md 
- https://github.com/webpack-contrib/style-loader/blob/master/CHANGELOG.md
- https://github.com/webpack/webpack/releases
- https://github.com/webpack/webpack-dev-server/releases

Also added `--open` flag to automatically open web browser for docs dev.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "build": "npm run build:tokens && npm run build:svgs && lerna run build && (cd ./native && npm run link)",
     "storybook": "start-storybook -p 9001",
     "sassdoc": "sassdoc {packages/bpk-mixins/src/**/*,packages/bpk-svgs/dist/*,packages/bpk-tokens/tokens/base.default}.scss -d dist/sassdoc -v --strict --debug",
-    "docs": "webpack-dev-server",
+    "docs": "webpack-dev-server --open",
```
Comment (matteo-hertel) [inline package.json]: Awesome!! 🎉

**Comment by matthewdavidson** [comment]
I've tested storybook dev (`npm start`), docs dev (`npm run docs`) and docs prod output (`npm run docs:dist && http-server dist`).

File: packages/bpk-stylesheets/base.js [hunk 2]
```diff
@@ -63,23 +60,17 @@
 /******/ 	__webpack_require__.p = "";
 /******/
 /******/ 	// Load entry module and return exports
-/******/ 	return __webpack_require__(__webpack_require__.s = 1);
+/******/ 	return __webpack_require__(__webpack_require__.s = 0);
```
Comment (shaundon) [inline packages/bpk-stylesheets/base.js]: I couldn't find in the links you posted where this change comes from - what does it do?

**Comment by matthewdavidson** [comment]
@k0nserv me too - for comparison, here is CRA's webpack 3 upgrade PR https://github.com/facebookincubator/create-react-app/pull/2574

File: packages/bpk-stylesheets/base.js (hunk 2 above)
Comment (matthewdavidson) [reply packages/bpk-stylesheets/base.js]: extract-text-webpack-plugin added sorting https://github.com/webpack-contrib/extract-text-webpack-plugin/blob/master/CHANGELOG.md#300-rc1-2017-07-07

**Pull request merged**